src/toolbox/build/index.ts

```diff
@@ -212,9 +214,14 @@ export async function build({
   if (mode === 'development') configArgs.push('-d')
   if (mode === 'production') configArgs.push('-i')
 
+  Object.keys(config).forEach(element => {
+    configArgs.push(`${element}="${config[element as keyof typeof config] as string}"`)
+  })
```

Comment:
Using [`Object.entries`](http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries), we can get the key and value as part of the `forEach` iterator:

```suggestion
  Object.entries(config).forEach(([element, value]) => {
    configArgs.push(`${element}="${value}"`)
  })
```